src/js/apps/forms/form/form_app.js

```diff
@@ -152,9 +152,21 @@ export default App.extend({
     this.responses.unshift(response);
     this.setState({ responseId: response.id });
   },
-  onFormServiceError() {
+  onFormServiceError(errors) {
     if (this.loadingModal) this.loadingModal.destroy();
 
+    if (errors[0].status === 403) {
+      this.isLocked = true;
+
+      this.showLockedSubmit();
+
+      this.getRegion('formUpdated').empty();
```

Comment:
Added this to avoid:

<img width="446" alt="Screenshot 2023-11-01 at 12 21 53 PM" src="https://github.com/RoundingWell/care-ops-frontend/assets/35355575/fcc8f1a7-0c47-47a5-96f8-c9e731a31a57">

My assumption is that we don't want to show those messages simultaneously.

Reply:
Well... but if the user keeps working in the form after the 403 error, that message will show. So I guess we'll need a `if (this.isLocked) return` check in the `showLastUpdated()` function.